src/app/store.ts

```diff
@@ -1,10 +1,20 @@
 import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
 // eslint-disable-next-line import/no-cycle
```

Comment:
add circle dependency it is bad practice, try to move some code into new files and avoid cycle dependency